src/VisualStudio/Core/Def/Implementation/Utilities/BrowserHelper.cs

```diff
@@ -59,7 +59,8 @@ private static bool TryStartBrowser(IServiceProvider serviceProvider, Uri uri)
 
         private static bool TryStartBrowser(IVsWebBrowsingService service, Uri uri)
         {
-            return ErrorHandler.Succeeded(service.CreateExternalWebBrowser((uint)__VSCREATEWEBBROWSER.VSCWB_ForceNew, VSPREVIEWRESOLUTION.PR_Default, uri.AbsoluteUri));
+            IVsWindowFrame unused;
+            return ErrorHandler.Succeeded(service.Navigate(uri.AbsoluteUri, 0, out unused));
         }
 
         private static void StartBrowser(Uri uri)
```

Comment:
Makes sense. 

I would be surprised if this ends up launching external browser without any error (especially once we have a setting for this somewhere and I set it to explicitly prefer internal browser). But I guess that's ok...